### ISSUE TITLE
Is the link to SVN still correct?

### ISSUE THREAD

**Issue by jernst** [dshanske/indieweb-post-kinds#148]
On https://wordpress.org/plugins/indieweb-post-kinds/#developers, it refers to an SVN repo here: https://plugins.svn.wordpress.org/indieweb-post-kinds/ . Is that still correct? There are some other link there, too, that may need updating.

**Comment by miklb** [comment]
the link *should* be correct as I believe when a new tag/release is created it runs a script that updates the SVN repo. But note that repo is only for tags, not current GitHub code, so not a direct mirror.

**Comment by dshanske** [comment]
WordPress doesn't let you change that. I have to commit plugins to their SVN repository. My readme says issues on GitHub

**Comment by jernst** [comment]
I never noticed that this is hard-coded. Some other plugins like https://wordpress.org/plugins/akismet/#developers cleverly hide it under the "Read more", so I didn't see it. Only meant to say that when I followed the SVN link and went to `trunk`, I found a code base that looked very different from what is in Git. Perhaps I would have found the correct version under `tags`?

I'll close this, sorry for the trouble.
